tests/test_readingorder.py

```diff
@@ -62,13 +62,16 @@ def test_order_simple_over_under(self):
 
             BBBB
         
+        The reading order should be the same for left-to-right and right-to-left.
```

Comment:
This isn't true. LTR/RTL should change the order of principal text blocks, e.g an Arabic page will be read right column before left column while a Latin text would be reverse.

Reply:
Thanks for commenting! I'm glad this test already sparks a discussion about the expectations of an important function :)

In this toy example, there are only two lines (no regions) and these lines are exactly one above the other. So I assumed only top-to-bottom ordering matters here. Is that a wrong assumption?

Reply:
Ah yeah, for stacked lines it doesn't matter. Columnar behavior should also, in principle, be independent of the regions. Regions *should* just aid that behavior by ensuring that the algorithm doesn't jump out of the regions for straying lines.